bedrock/base/templates/base-protocol-mozilla.html

```diff
@@ -10,6 +10,9 @@
 
 {% block site_css %}
   {{ css_bundle('protocol-mozilla') }}
+  {% if switch('m24-footer') %}
+    {{ css_bundle('footer-refresh') }}
```

Comment:
I'd also like for us to try and reach a resolution on the issue highlighted here before merging.

Reply:
I am in conversation with Stephanie about this. Might take a few hours due to the time zone differences.

Reply:
We also only use the font locally still. Shall we also update the fonts in this PR or it is okay to open a second PR for font updates?

Reply:
Yes, we may have a scenario where we want to disable the header for footer, but not the other. Or a scenario where the header is completely translated but not the footer yet.

I'm hesitant to have a separate switch for the CSS in case it gets toggled off while the HTML is still on but I think we can have a single bundle and include it if either switch is active.

Reply:
I'm not sure I quite understand what you're saying. But all I ask is that we weight up these (unclear, at least to me) scenarios with performance of our website (in terms of how much redundant CSS / JS we're serving our visitors, and how many fonts we're loading). 

We could always have logic that say if either the nav or footer aren't ready for a particular locale, then show neither of them.

Reply:
Okay, @wen-2018 please do what Alex is suggesting.